browser/ui/views/brave_help_bubble/brave_help_bubble_delegate_view.cc

```diff
@@ -204,6 +205,8 @@ void BraveHelpBubbleDelegateView::SetUpLabel(views::Label* label,
   label->SetMultiLine(true);
   label->SetMaximumWidth(390);
   label->SetText(text);
+  label->SetEnabledColor(SK_ColorWHITE);
```

Comment:
We need to explicitly set the color because on Windows:

![image](https://user-images.githubusercontent.com/8665427/216146879-4eb11c17-c4e1-4077-9525-57d4a33ef55f.png)

Reply:
good find. we usually need to do that, and sometimes we also need to call SetAutoredabilityColor(false) when we don't want automatic color adjustment based on background color. 

https://source.chromium.org/chromium/chromium/src/+/012eae9e445b70442b2ff96bc852b717ebe08d89:ui/views/controls/label.h;l=128

Reply:
Thanks